src/types/index.ts

```diff
@@ -187,3 +187,38 @@ export interface MarketSnapshot {
   totalSupply: string;
   updatedAt: string;
 }
+
+export type TransactionEvents =
+  | 'Mint'
+  | 'Transfer'
+  | 'Borrow'
+  | 'RepayBorrow'
+  | 'Redeem'
+  | 'Approval'
+  | 'LiquidateBorrow'
+  | 'ReservesAdded'
+  | 'ReservesReduced'
+  | 'MintVAI'
+  | 'Withdraw'
+  | 'RepayVAI'
+  | 'Deposit'
+  | 'VoteCast'
+  | 'ProposalCreated'
+  | 'ProposalQueued'
+  | 'ProposalExecuted'
+  | 'ProposalCanceled';
+
+export interface Transaction {
+  amount: BigNumber;
+  blockNumber: number;
+  category: 'vtoken' | 'vai' | 'vote';
+  createdAt: Date;
+  event: TransactionEvents;
+  from: string;
+  id: number;
+  timestamp: string | null;
+  to: string;
+  transactionHash: string;
+  updatedAt: string;
+  vTokenAddress: string;
```

Comment:
Can we get the vToken ID too, or does the backend/smart contracts only return the address?

Reply:
This is returned from the backend api. We could look at updating the payload

Reply:
Got you 👍
I think for now let's leave it as is